src/main/java/org/manifold/compiler/back/digital/smt2/Symbol.java

```diff
@@ -0,0 +1,67 @@
+package org.manifold.compiler.back.digital.smt2;
+
+import java.io.IOException;
+import java.io.Writer;
+
+public class Symbol extends SExpression {
+  // TODO symbol names do not always need to be escaped
```

Comment:
I'm going to remove this, since I'm pretty sure we will never generate an invalid symbol name here.